lib/gs-utils.c

```diff
@@ -1059,4 +1059,13 @@ gs_utils_update_install_timestamp (GSettings *settings)
 	g_settings_set (settings, "install-timestamp", "x", g_date_time_to_unix (now));
 }
 
+gboolean
+gs_utils_app_is_auto_updating (GsApp *app)
```

Comment:
In the commit message, make sure you write the reason for this move (that it will be used later in different parts of the code).